guides/coding_style/c++/style.md

```diff
@@ -70,6 +73,14 @@ public:
 */
 ```
 
+Namespace
+===
+When using namespace, please pay attention to `name collision`(also `name conflict`):
```

Comment:
Remove the please, and reword to name collision **_s_**. Also remove the code tags are name collision/name conflict.